cgrok/grokre.c

```diff
@@ -217,7 +219,7 @@ static char *grok_pattern_expand(grok_t *grok) {
       if (definition_len > 0) {
         /* We got an in-line definition */
         /* discard const-ness with reckless abandon! */
-        pattern_regex = strndup(full_pattern + capture_vector[g_cap_definition * 2], definition_len);
+        pattern_regex = string_ndup(full_pattern + capture_vector[g_cap_definition * 2], definition_len);
```

Comment:
This is from the Github master, Windows doesn't have `strndup` so they reimplemented it as `string_ndup`. It's part of compilation and not on the critical path so I'm not as worried about speed.